db/migrate/20170109115157_fix_lookup_key_auditable_type.rb

```diff
@@ -0,0 +1,15 @@
+class FixLookupKeyAuditableType < ActiveRecord::Migration
+  def up
+    Audit.reorder(nil).joins('JOIN lookup_keys ON lookup_keys.id = audits.auditable_id')
+         .where(:auditable_type => 'LookupKey', :lookup_keys => {:type => 'VariableLookupKey'})
+         .update_all(:auditable_type => 'VariableLookupKey')
+
+    Audit.reorder(nil).joins('JOIN lookup_keys ON lookup_keys.id = audits.auditable_id')
+         .where(:auditable_type => 'LookupKey', :lookup_keys => {:type => 'PuppetclassLookupKey'})
+         .update_all(:auditable_type => 'PuppetclassLookupKey')
```

Comment:
tried to get the two queries into one with `update_all('auditable_type = lookup_keys.type')` but: https://github.com/rails/rails/issues/13496